specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-01-01-preview/examples/Schedule/trigger.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "test-rg",
+    "workspaceName": "my-aml-workspace",
+    "name": "string",
+    "api-version": "2024-01-01-preview",
+    "body": {
+      "scheduleTime": "string"
```

Comment:
Could you please update it to a more meaningful value?

Reply:
Updated.